cmake/modules/RootBuildOptions.cmake

```diff
@@ -102,37 +102,37 @@ ROOT_BUILD_OPTION(cxx14 OFF "Build using C++14 compatible mode, requires gcc > 4
 #ROOT_BUILD_OPTION(cxx17 OFF "Build using C++17 compatible mode, requires gcc >= 7.1.0 or clang")
 ROOT_BUILD_OPTION(cxxmodules OFF "Compile with C++ modules enabled.")
 ROOT_BUILD_OPTION(davix ON "DavIx library for HTTP/WEBDAV access")
-ROOT_BUILD_OPTION(dcache ON "dCache support, requires libdcap from DESY")
+ROOT_BUILD_OPTION(dcache OFF "dCache support, requires libdcap from DESY")
 ROOT_BUILD_OPTION(exceptions ON "Turn on compiler exception handling capability")
 ROOT_BUILD_OPTION(explicitlink ON "Explicitly link with all dependent libraries")
 ROOT_BUILD_OPTION(fftw3 ON "Fast Fourier Transform support, requires libfftw3")
-ROOT_BUILD_OPTION(fitsio ON "Read images and data from FITS files, requires cfitsio")
+ROOT_BUILD_OPTION(fitsio OFF "Read images and data from FITS files, requires cfitsio")
 ROOT_BUILD_OPTION(fortran ON "Enable the Fortran components of ROOT")
 set(gcctoolchain "" CACHE PATH "Path for the gcctoolchain in case not the system gcc is used to build clang/LLVM")
 ROOT_BUILD_OPTION(gdml OFF "GDML writer and reader")
```

Comment:
This would be a nice thing to have `ON` by default, actually.

Reply:
Why?

Reply:
It's lightweight and used by people to read geometry files for simulations, visualization, etc. There is no reason to distribute ROOT with this turned OFF (if we distribute it with default options).

Reply:
I think (see also Davix I guess) that this was held back by requiring an external library (xml parsing I think) ... It would indeed be nice to have it on by default.

Reply:
I think we should propose people to write geometries in ROOT format rather than build them on the fly starting from configuration files. Said that, I trust you if you say a community which is big enough would benefit from this wrt the size of the burden of compiling this every time by default.

Reply:
GDML files are sometimes passed around by the experiments, it's an easy format to ingest a geometry into ROOT. Most people will use a file created by someone else I would say, and it's also an easy way to exchange geometries with Geant4.